### PR TITLE
Broken links removed and markdown style fixed with plugin.

### PR DESCRIPTION
**Pull request by KaganCanSit**
Hello,
Considering issue numbered "#457" titled "Dead links", I searched for broken links in the repository. I tried to edit it as much as I could and replace it with updated links, if any.

It was a little sad to see that the "https://turkcekaynaklar.com" site is not live at the moment. I believe that such a project aimed at highlighting the Turkish language will be in better places. I also have a few specific ideas about the content of the project. We can make the content more readable and equipped with visual content. I can help with this.

Since I check the links with a script I wrote, there must be some I missed. If there are parts that bother you or that you think are incorrect, please contact me. I will try to deal with it as soon as possible.

I'm adding the links and information I found below. Best regards.

---

I removed the link under. The site is currently in service. But I couldn't find any content from 2017 or with the same title.
http://www.ilkaygunel.com/blog/2017/angular-2-tutorials/

The site appears to be under maintenance.
https://www.yazilimgunlugu.org/category/javascript-web-uygulama-gelistirme/angular-2-4-turkce/

[ERROR] [LINK UNKNOWN STATUS CODE (410)] - https://docs.google.com/forms/d/e/1FAIpQLSdeuvyiBvhP46os0xhPJE4mnTgtHfWpfCp-595VwbES9Qpdfg/viewform
        File: ./README.md
[ERROR] [LINK CURL ERROR 52] - http://devpod.org/
        File: ./README.md
[ERROR] [LINK COULD NOT RESOLVE HOST] - http://atakanermis.tk
        File: ./README.md
[ERROR] [LINK COULD NOT RESOLVE HOST] - https://www.getrevue.co/profile/ai
        File: ./README.md
[ERROR] [LINK NOT FOUND] - https://github.com/vimevim/turkce-C-sharp-kaynaklari
        File: ./README.md
[ERROR] [LINK NOT FOUND] - http://www.bahadirakin.com/category/programlama/java/
        File: ./README.md
[ERROR] [LINK COULD NOT RESOLVE HOST] - https://kodpod.live/
        File: ./README.md
[ERROR] [LINK FAILED TO CONNECT TO HOST] - https://turkcekaynaklar.com
        File: ./README.md
[ERROR] [LINK COULD NOT RESOLVE HOST] - http://acikkaynak.info
        File: ./README.md
[ERROR] [LINK NOT FOUND] - http://ozaksut.com/kategori/xamarin/
        File: ./README.md
[ERROR] [LINK NOT FOUND] - https://www.spreaker.com/show/birsifir-podcast
        File: ./README.md
[ERROR] [LINK NOT FOUND] - https://github.com/vimevim/turkce-c-kaynaklari
        File: ./README.md
[ERROR] [LINK COULD NOT RESOLVE HOST] - https://www.getrevue.co/profile/etaylan
        File: ./README.md
[ERROR] [LINK NOT FOUND] - http://bonsaiden.github.io/JavaScript-Garden/tr/
        File: ./README.md
[ERROR] [LINK SERVICE UNAVAILABLE] - http://rubytr.herokuapp.com/
        File: ./README.md
[ERROR] [LINK NOT FOUND] - https://podcasts.apple.com/tr/podcast/turuncu-pasaport/id1435681904
        File: ./README.md
[ERROR] [LINK NOT FOUND] - https://emredurukn.github.io/2018/11/09/ruby-ile-web-scrapinge-giris.html
        File: ./README.md
[ERROR] [LINK COULD NOT RESOLVE HOST] - http://www.giik.fm
        File: ./README.md
[ERROR] [LINK NOT FOUND] - https://www.spreaker.com/show/parasutle-uretim-bandi
        File: ./README.md
[ERROR] [LINK NOT FOUND] - https://www.mkoseoglu.com/linux-manjaro-gnome-3-22-cozumleri
        File: ./README.md
[ERROR] [LINK COULD NOT RESOLVE HOST] - https://hurriyetlabs.com/react-v16-2-0-ile-iyile%C5%9Ftirilen-%C3%A7ok-k%C4%B1ymetli-%C3%B6zellik-fragments-7a757094eafe
        File: ./README.md
[ERROR] [LINK COULD NOT RESOLVE HOST] - https://www.acikakademi.com/portal/egitimler/aspnet-mvc-ile-ileri-seviyede-web-programlama.aspx
        File: ./README.md
[ERROR] [LINK UNKNOWN STATUS CODE (405)] - https://umuts.info
        File: ./README.md
[ERROR] [LINK SERVICE UNAVAILABLE] - https://maker.robotistan.com/kategori/arduino/
        File: ./README.md
[ERROR] [LINK NOT FOUND] - https://github.com/vimevim/turkce-cpp-kaynaklari
        File: ./README.md
[ERROR] [LINK COULD NOT RESOLVE HOST] - https://www.getrevue.co/profile/mhkoca
        File: ./README.md
[ERROR] [LINK NOT FOUND] - https://img.shields.io/travis/fatihacet/turkcekaynaklar-com.svg?style=flat-square
        File: ./README.md
[ERROR] [LINK COULD NOT RESOLVE HOST] - https://hurriyetlabs.com/javascript-hap-yaz%C4%B1s%C4%B1-ve-aras%C4%B1ndaki-farklar-javascriptte-e%C5%9Fitlik-b7cd20f91a
        File: ./README.md
[ERROR] [LINK NOT FOUND] - https://acikkaynak-slack-inviter.herokuapp.com/
        File: ./README.md
[ERROR] [LINK NOT FOUND] - https://github.com/p1v0t/VimKisayollari
        File: ./README.md
[ERROR] [LINK UNKNOWN STATUS CODE (410)] - http://www.belgeler.org/bashref/bashref.html
        File: ./README.md
[ERROR] [LINK NOT FOUND] - https://cliexperiences.wordpress.com/2017/08/01/whatsapp-altyapisina-teknik-bir-bakis/
        File: ./README.md
[ERROR] [LINK COULD NOT RESOLVE HOST] - https://www.ilkayilknur.com/aspnet-core-uygulamalarinda-ihttpclientfactory-kullanimi
        File: ./README.md
[ERROR] [LINK NOT FOUND] - https://pyistanbul.herokuapp.com/
        File: ./README.md
[ERROR] [LINK COULD NOT RESOLVE HOST] - https://www.acikakademi.com/portal/egitimler/csharp-uygulama-gelistirme.aspx
        File: ./README.md
[ERROR] [LINK INTERNAL SERVER ERROR] - https://go-tour-turkish.appspot.com/welcome/1
        File: ./README.md
[ERROR] [SSL HANDSHAKE FAILED] - https://blog.kodcu.com
        File: ./README.md
[ERROR] [LINK UNKNOWN STATUS CODE (405)] - http://umuts.info/backend-best-practices/
        File: ./README.md
[ERROR] [LINK CURL ERROR 56] - http://www.lojikprob.com/c-ile-avr-programlama-e-kitap-indir/
        File: ./README.md
[ERROR] [LINK NOT FOUND] - https://github.com/umutphp/html-best-practices/blob/master/README.tr.md:
        File: ./README.md
[ERROR] [LINK NOT FOUND] - https://www.girisimcimuhabbeti.com/
        File: ./README.md
[ERROR] [SSL HANDSHAKE FAILED] - https://blog.kodcu.com/2012/06/java-pass-by-value/
        File: ./README.md
[ERROR] [LINK COULD NOT RESOLVE HOST] - http://www.injavawetrust.com/
        File: ./README.md
[ERROR] [LINK COULD NOT RESOLVE HOST] - https://hurriyetlabs.com/javascript-hap-yaz%C4%B1s%C4%B1-element-insertadjacenthtml-8fe1e6511808
        File: ./README.md
[ERROR] [LINK UNKNOWN STATUS CODE (410)] - http://www.belgeler.org/glibc/glibc.html
        File: ./README.md
[ERROR] [LINK NOT FOUND] - https://cagatay.me/nodejs-t%C3%BCrk%C3%A7e-e%C4%9Fitim-seti-giri%C5%9F-1-815b6fd21502
        File: ./README.md
[ERROR] [LINK UNKNOWN STATUS CODE (405)] - http://umuts.info/hacker-laws-tr/
        File: ./README.md
[ERROR] [LINK NOT FOUND] - https://gokmengorgen.net/django-notlarim/
        File: ./README.md
[ERROR] [LINK NOT FOUND] - http://ozcanacar.com/
        File: ./README.md
[ERROR] [LINK UNKNOWN STATUS CODE (410)] - https://www.mobiler.dev/android
        File: ./README.md
[ERROR] [LINK COULD NOT RESOLVE HOST] - http://www.xamarinturkiye.com/
        File: ./README.md
[ERROR] [LINK COULD NOT RESOLVE HOST] - https://jargon.ist/
        File: ./README.md
[ERROR] [LINK COULD NOT RESOLVE HOST] - https://www.arduinoprojeler.com/
        File: ./README.md
[ERROR] [LINK CURL ERROR 56] - http://www.lojikprob.com/diger/kitap/avr-ile-mikrodenetleyici-uygulamalari-gokhan-dokmetas-ucretsiz-e-kitap/
        File: ./README.md

### CODE REVIEW

**Comment by gokaygurcan** [comment]
😱

**Comment by KaganCanSit** [comment]
Hello @gokaygurcan ,
I am aware that a pull request of this size is inappropriate and scary for the open source world. However, the purpose of this repository is to provide resources by guiding people and I think I should evaluate the link contents as a whole. I thought it would be more costly to fix the connections gradually.

Best regards.

File: README.md [hunk 1]
```diff
@@ -1,89 +1,88 @@
-## Türkçe Kaynaklar Listesi [![Build Status](https://img.shields.io/travis/fatihacet/turkcekaynaklar-com.svg?style=flat-square)](https://travis-ci.org/fatihacet/turkcekaynaklar-com)
+## Türkçe Kaynaklar Listesi
 
 Yazılım geliştirme ile ilgili özenle seçilmiş Türkçe kaynaklar listesi.
 
-### [https://turkcekaynaklar.com](https://turkcekaynaklar.com)
```
Comment (gokaygurcan) [inline README.md:5]: | Expires On | Registered On | Updated On |
|--------------|-------------|--------------|
| 2025-02-27 | 2017-02-27 | 2024-04-09 |

@fatihacet ?

File: README.md [hunk 2]
```diff
@@ -1,89 +1,88 @@
...
+  - [Android](#android)
+  - [Angular](#angular)
+  - [ASP.NET](#aspnet)
+  - [Arduino](#arduino)
+  - [Bash](#bash)
+  - [Bilgisayar Bilimleri](#bilgisayar-bilimleri)
+  - [Blockchain](#blockchain)
+  - [Bot Framework](#bot-framework)
+  - [C](#c)
+  - [Csharp](#csharp)
```
Comment (gokaygurcan) [inline README.md:19]: image

File: README.md [hunk 3]
```diff
@@ -1,89 +1,88 @@
...
+  - [CSS](#css)
+  - [D](#d)
+  - [Dart](#dart)
+  - [Deep Learning](#deep-learning)
+  - [Design Patterns](#design-patterns)
+  - [Django](#django)
+  - [Docker](#docker)
+  - [Electron](#electron)
+  - [Flux](#flux)
+  - [Flutter](#flutter)
```
Comment (gokaygurcan) [inline README.md:33]: T before X?

File: README.md [hunk 4]
```diff
@@ -1,89 +1,88 @@
...
+  - [Haskell](#haskell)
+  - [HTML](#html)
+  - [Ionic](#ionic)
+  - [Java](#java)
+  - [JavaScript](#javascript)
+  - [jQuery](#jquery)
+  - [Kotlin](#kotlin)
+  - [Laravel](#laravel)
+  - [Machine Learning](#machine-learning)
+  - [Matlab](#matlab)
```
Comment (gokaygurcan) [inline README.md:49]: https://en.wikipedia.org/wiki/MATLAB

File: README.md [hunk 5]
```diff
@@ -1,89 +1,88 @@
...
+  - [PHP](#php)
+  - [Postgres](#postgres)
+  - [PowerShell](#powershell)
+  - [Python](#python)
+  - [Rails](#rails)
+  - [React](#react)
+  - [React Native](#react-native)
+  - [Ruby](#ruby)
+  - [Rust](#rust)
+  - [RXJS](#rxjs)
```
Comment (gokaygurcan) [inline README.md:64]: https://rxjs.dev/

**Comment by KaganCanSit** [comment]
Hello @gokaygurcan, 
I am interested in feedback immediately. If you are available, we can review it immediately in a discord session.

File: README.md (hunk 1 above)
Comment (KaganCanSit) [reply README.md:5]: I really couldn't understand this situation. VPN etc. to site content. I cannot access it with additional tools. I think there may be a problem with Deploy.

File: README.md (hunk 3 above)
Comment (KaganCanSit) [reply README.md:33]: The "Markdown All in One" plugin caused this. I'm fixing it.

File: README.md [hunk 6]
```diff
@@ -1,89 +1,88 @@
...
+  - [Sketch](#sketch)
+  - [Solidity](#solidity)
+  - [Sourcemod](#sourcemod)
+  - [Spring Boot](#spring-boot)
+  - [SQL](#sql)
+  - [Swift](#swift)
+  - [Titanium Mobile](#titanium-mobile)
+  - [TypeScript](#typescript)
+  - [Vault](#vault)
+  - [Veri Bilimi (Data Science)](#veri-bilimi-data-science)
```
Comment (gokaygurcan) [inline README.md:74]: New section? Please move into its own PR.

File: README.md [hunk 7]
```diff
@@ -1,89 +1,88 @@
...
+  - [Vue.js](#vuejs)
+  - [WordPress](#wordpress)
+  - [Xamarin](#xamarin)
+- [Diğer Kaynaklar](#diğer-kaynaklar)
   - [Faydalı Linkler](#faydalı-linkler)
   - [Podcast Yayınları](#podcast-yayınları)
   - [YouTube Kanalları](#youtube-kanalları)
   - [Twitch Kanalları](#twitch-kanalları)
   - [Slack Grupları](#slack-grupları)
+  - [Discord Grupları](#discord-grupları)
```
Comment (gokaygurcan) [inline README.md:85]: New section? Please move into its own PR.

File: README.md (hunk 4 above)
Comment (KaganCanSit) [reply README.md:49]: I corrected the naming. (MATLAB)

File: README.md (hunk 5 above)
Comment (KaganCanSit) [reply README.md:64]: I corrected the naming as "RxJs".

**Review by gokaygurcan — CHANGES_REQUESTED** [review]
Cheers

File: README.md (hunk 2 above)
Comment (KaganCanSit) [reply README.md:19]: Fixing. Copilot failed.

File: README.md (hunk 6 above)
Comment (KaganCanSit) [reply README.md:74]: There was a distinction in the link section below for this. But there was no go link in the category list. Actually, it was not newly added.

File: README.md (hunk 7 above)
Comment (KaganCanSit) [reply README.md:85]: There was a distinction in the link section below for this. But there was no go link in the category list. Actually, it was not newly added. Main repo line 907.

File: README.md [hunk 8]
```diff
@@ -1,89 +1,88 @@
...
-- [Vim](#vim)
-- [Vue.js](#vuejs)
-- [WordPress](#wordpress)
-- [Xamarin](#xamarin)
-- Diğer Kaynaklar
+### Konu Başlıkları
+  - [Android](#android)
+  - [Angular](#angular)
+  - [ASP.NET](#aspnet)
+  - [Arduino](#arduino)
```
Comment (gokaygurcan) [inline README.md:13]: Hmm, I missed this one. R before S.

File: README.md (hunk 6 above)
Comment (gokaygurcan) [reply README.md:74]: 👍

File: README.md (hunk 1 above)
Comment (gokaygurcan) [reply README.md:5]: I'll ask him next time we see each other. We can update or put it back accordingly afterwards. Marking as resolved for now.

File: README.md (hunk 8 above)
Comment (KaganCanSit) [reply README.md:13]: Actually, the main problem is that the order of the contents is not correct. I did it by looking at the content. I have to fix them too.

Comment (gokaygurcan) [reply README.md:13]: 🤷‍♂️

File: README.md (hunk 3 above)
Comment (KaganCanSit) [reply README.md:32]: It is also wrong during the main content. I'm fixing it.

File: README.md (hunk 4 above)
Comment (KaganCanSit) [reply README.md]: The main content title is also small, I am changing it.

**Comment by KaganCanSit** [comment]
There was rapid progress. 💯(Do not write code while listening to AC/DC. :D) I left the ones that had a title in the content but did not have a jump link in the list as added. Another thing?

**Review by gokaygurcan — APPROVED** [review]
Let's gooooo!

**Comment by gokaygurcan** [comment]
Thanks!

**Comment by KaganCanSit** [comment]
@gokaygurcan You are welcome, it was fun. If there is anything I can do, just a notification to kagancansit@hotmail.com is sufficient.

**Comment by gokaygurcan** [comment]
Sure, thanks a lot. If you see some interesting articles and stuff, feel free to add them here ^_^
